vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +137,49 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            locator.getSearchParameters().setWaitForElement(false);
+            IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>() {
+                private final List<Point> points = new ArrayList<>();
+                private Point lastPosition;
+
+                @Override
+                public Boolean apply(SearchContext searchContext)
+                {
+                    try
+                    {
+                        return searchActions.findElement(context, locator).map(WebElement::getLocation).map(s ->
+                        {
+                            boolean samePosition = s.equals(lastPosition);
```

Comment:
why not to check last element in points collection instead of having lastPosition variable?